testdata/go/v4-alpha/monitoring/memcached-operator/monitoring/alerts.go

```diff
@@ -0,0 +1,84 @@
+package monitoring
+
+import (
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	ruleName                     = "memcached-operator-rules"
+	alertRuleGroup               = "memcached.rules"
+	deploymentSizeUndesiredAlert = "MemcachedDeploymentSizeUndesired"
+	operatorDownAlert            = "MemcachedOperatorDown"
+	operatorUpTotalRecordingRule = "memcached_operator_up_total"
+	runbookURLBasePath           = "https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v4-alpha/monitoring/memcached-operator/runbooks/"
```

Comment:
Will this be hardcoded in operator-sdk generator or are you thinking about configuring it somehow?

Reply:
@machadovilaca not sure I understood exactly what you are referring. Please explain more if possible.

Reply:
every new operator created will have a different runbookURLBasePath, right? My question was if you intended to leave this as `https://github.com/operator-framework/operator-sdk/tree...` by default and let the user change that later, or if you create it on operator code generation time

Reply:
Not sure what it has to change. The samples are stable on Github, no?
This code is only for the monitoring operator sample which will always have `memcached-operator` name, and its runbook base URL will always be `https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v4-alpha/monitoring/memcached-operator/runbooks/`. 
But please correct me if I'm wrong.

Reply:
> every new operator created will have a different runbookURLBasePath

Why is that?

Reply:
in testdata yes, it won't change, my question is more for when you do the generator.

Reply:
the generator is also for the samples, so don't see why this URL should be generated differently than other parts.
Basically, I planned to generate it hardcoded

Reply:
that generator is what operator-sdk uses to create new operator projects. What will those operators have in `runbookURLBasePath`?

Reply:
AFAIK, the generator we're working on is only for the samples on testdata, which are not scafold by the default  operator-sdk scaffolding, no? 
https://github.com/operator-framework/operator-sdk/blob/master/hack/generate/samples/generate_testdata.go#L64

Reply:
mhh, maybe, but then users won't have the monitoring stuff if they generate a new project, right?

Reply:
@assafad @machadovilaca So my understanding on this PR is specifically to add the alerts and corresponding code *only* on the testdata. We have a generator specifically for generating our testdata that @assafad is updating and not the actual generation logic used in the `operator-sdk` cli tool that users would use to scaffold their projects.

That being said, I think @machadovilaca points do raise the question - If the plan is to ultimately create a plugin that will do most of the heavy lifting by scaffolding most of this logic, should we really be updating the testdata prior to creating that plugin? It feels to me like it may be better to go ahead and start writing the plugin and tackle updating the testdata once the plugin is written so you aren't writing testdata generation logic just to delete it later when the plugin automatically scaffolds *most* of that logic for you.

Reply:
we actually just met to discuss this, and we agree with you, it does not make sense to update the testdata generator, but to first create a plugin in kubebuilder instead